<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageStageAccessoriesWgtClass</class>
 <widget class="QWidget" name="PageStageAccessoriesWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Magnetic brake</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="6" column="1">
       <widget class="QCheckBox" name="mbAvailableCheck">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Torque:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="FloatSpinBox" name="mbRatedCurrentSpin">
        <property name="suffix">
         <string> A</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Rated current:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Information:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="varlengthLineEdit" name="magneticBrakeInfoEdit">
        <property name="maxLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Available:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="FloatSpinBox" name="mbRatedVoltageSpin">
        <property name="suffix">
         <string> V</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Rated voltage:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Powered hold:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="mbPoweredHoldCheck">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="FloatSpinBox" name="mbTorqueSpin">
        <property name="suffix">
         <string> mN m</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Temperature sensor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="FloatSpinBox" name="tsMinSpin">
           <property name="suffix">
            <string> °C</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string> - </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="FloatSpinBox" name="tsMaxSpin">
           <property name="suffix">
            <string> °C</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Temp. range:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Information:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="varlengthLineEdit" name="temperatureSensorInfoEdit">
        <property name="maxLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Temp. gradient:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="FloatSpinBox" name="tsGradSpin">
        <property name="suffix">
         <string> V / °C</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="tsTypeCombo">
        <item>
         <property name="text">
          <string>Thermocouple</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Semiconductor</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Available:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="tsAvailableCheck">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Limit switches</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QCheckBox" name="lsOnSw1AvailableCheck">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>SW1 available:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>SW2 available:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>SW1 low is active:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>SW2 low is active:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_20">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Limit switches shorted:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="lsOnSw2AvailableCheck">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="lsSw1ActiveLowCheck">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="lsSw2ActiveLowCheck">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="lsShortedCheck">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tsMinSpin</tabstop>
  <tabstop>tsMaxSpin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
